### DISCUSSION TITLE
ESLint 8 for less - The "syntax" option is no longer available

### DISCUSSION THREAD

**Discussion by PlayApple** [eslint/eslint#15224] [Q&A] [answered]
> stylelint --fix "src/**/*.less" --syntax less
Error: The "syntax" option is no longer available. You should install an appropriate syntax, e.g. postcss-scss, and use the "customSyntax" option

`
"lint:style": "stylelint --fix \"src/**/*.less\" --syntax less",
`

How to use customSyntax to fixed it? Thanks

**Comment by aladdin-add** [comment, marked as answer]
I did not get your point. as stylelint, you'll need to talk to its maintainers (not maintained by the eslint team).

**Reply by PlayApple** [reply]
lol, I made a mistake, I have sent it to a stylelint issue.
